src/gatsby/src/components/advisories/advisoryCard.js

```diff
@@ -101,6 +143,31 @@ const AdvisoryCard = ({ advisory, index }) => {
                           {fireCentre.fireCentreName}
                         </Link>
                       ))}
```

Comment:
I think we need to display the fireZone badges too

Reply:
Oh you are right. It's not in the annotations, but there is an example that shows "Arrow fire zone"